packages/plugin-aws-elasticsearch/src/utils/AmazonConnection.js

```diff
@@ -0,0 +1,44 @@
+const { Connection } = require('@elastic/elasticsearch')
```

Comment:
please insert license from the package this code was taken (copy paste at the top of the file in the comments section)

Reply:
also suggest refactoring this into TS, should be quick